internal/test/environment/testenv.go

```diff
@@ -108,3 +109,15 @@ func SkipIfNotPlatform(t *testing.T, platform string) {
 	daemonPlatform := strings.TrimSpace(result.Stdout())
 	skip.If(t, daemonPlatform != platform, "running against a non %s daemon", platform)
 }
+
+func SkipIfAPIOlder(t *testing.T, minimumVersion string) {
+	t.Helper()
+	// Use Client.APIVersion instead of Server.APIVersion.
```

Comment:
Wondering if we should try to make the UX a bit similar to what we have in docker/docker; https://github.com/moby/moby/blob/a60546b084eef87910345876e32c3db46e7c3bbc/integration-cli/docker_cli_build_test.go#L3338

https://github.com/moby/moby/blob/a60546b084eef87910345876e32c3db46e7c3bbc/testutil/environment/environment.go#L131

Reply:
Done and extracted to https://github.com/docker/cli/pull/4873